arcgis-ios-sdk-samples/Geometry/Buffer list/BufferListViewController.swift

```diff
@@ -0,0 +1,238 @@
+            mapView.graphicsOverlays.addObjects(from: [boundaryGraphicsOverlay, bufferGraphicsOverlay, tappedLocationsGraphicsOverlay])
+            mapView.touchDelegate = self
+        }
+    }
+    
+    // MARK: Instance properties
+    
+    /// An image layer as the base layer of the map.
+    let mapImageLayer = AGSArcGISMapImageLayer(url: URL(string: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/USA/MapServer")!)
+    /// A polygon that represents the valid area of use for the spatial reference.
+    let boundaryPolygon: AGSPolygon = {
+        let boundaryPoints = [
+            AGSPoint(x: -103.070, y: 31.720, spatialReference: .wgs84()),
+            AGSPoint(x: -103.070, y: 34.580, spatialReference: .wgs84()),
+            AGSPoint(x: -94.000, y: 34.580, spatialReference: .wgs84()),
+            AGSPoint(x: -94.00, y: 31.720, spatialReference: .wgs84())
+        ]
+        let polygon = AGSPolygon(points: boundaryPoints)
+        let statePlaneNorthCentralTexas = AGSSpatialReference(wkid: 32038)!
+        return AGSGeometryEngine.projectGeometry(polygon, to: statePlaneNorthCentralTexas) as! AGSPolygon
+    }()
+    
+    /// An overlay to display the boundary.
+    var boundaryGraphicsOverlay: AGSGraphicsOverlay {
+        let overlay = AGSGraphicsOverlay()
+        let lineSymbol = AGSSimpleLineSymbol(style: .dash, color: .red, width: 5)
+        let boundaryGraphic = AGSGraphic(geometry: boundaryPolygon, symbol: lineSymbol)
+        overlay.graphics.add(boundaryGraphic)
+        return overlay
+    }
+    /// An overlay to display buffers graphics.
+    let bufferGraphicsOverlay: AGSGraphicsOverlay = {
+        let overlay = AGSGraphicsOverlay()
+        let bufferPolygonOutlineSymbol = AGSSimpleLineSymbol(style: .solid, color: .systemGreen, width: 3)
+        let bufferPolygonFillSymbol = AGSSimpleFillSymbol(style: .solid, color: UIColor.yellow.withAlphaComponent(0.6), outline: bufferPolygonOutlineSymbol)
+        overlay.renderer = AGSSimpleRenderer(symbol: bufferPolygonFillSymbol)
+        return overlay
+    }()
+    /// An overlay to display tapped locations with red circle symbols.
+    let tappedLocationsGraphicsOverlay: AGSGraphicsOverlay = {
+        let overlay = AGSGraphicsOverlay()
+        let circleSymbol = AGSSimpleMarkerSymbol(style: .circle, color: .red, size: 10)
+        overlay.renderer = AGSSimpleRenderer(symbol: circleSymbol)
+        return overlay
+    }()
+    /// An array of tapped points and buffer radii (in US feet) tuple.
+    var tappedPointsAndRadius = [(point: AGSPoint, radius: Double)]() {
+        didSet {
+            undoBarButtonItem.isEnabled = !tappedPointsAndRadius.isEmpty
+            clearBarButtonItem.isEnabled = !tappedPointsAndRadius.isEmpty
+            // Redraw the buffers.
+            drawBuffers()
+        }
+    }
+    
+    /// The radius of the buffer.
+    var bufferRadius: Measurement<UnitLength> = Measurement(value: 0, unit: .miles)
```

Comment:
Correct me if I'm wrong, but it seems like this property isn't really necessary. The unit is used in one place, but the unit is always the same, so you could either declare a constant for it or just use `UnitLength.miles` directly. The value is only referenced in `doneAction`'s handler, so there really isn't a reason to store it in a property.